src/pages/index.js

```diff
@@ -30,6 +30,7 @@ import {
 import PopupWithImage from "../components/PopupWithImage.js";
 import FormValidator from "../components/FormValidator.js";
 import Api from "../utils/API.js";
+import { MAX_SAFE_INTEGER } from "core-js/core/number";
 const api = new Api({
   baseUrl: "https://around.nomoreparties.co/v1/group-12",
```

Comment:
I don't see any changes from the project 9 in this pull request. You should create a pull request with project 9 functionality and send it for the review. (There should not be project 9 functionality in the main branch while the pull request is not approved)